src/extend-context-menu.ts

```diff
@@ -4,7 +4,7 @@ import copyCellValue from "./table/utils/copy-utils";
 
 export default function extendContextMenu() {
   onContextMenu?.((menu: any, event: Event) => {
-    const target = event.target as HTMLElement;
+    const target = event.target as HTMLElement | null;
```

Comment:
Since this is called from external code (Sense?), it does seem like really good idea to guard against missing target here. Might even want to do `event?.target...`